src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -351,4 +351,41 @@
       it { expect { call_sourcediff }.to raise_error(ActionController::RoutingError, 'Expected AJAX call') }
     end
   end
+
+  describe 'GET #index' do
+    let!(:bs_requests) do
+      3.times do
+        create(:bs_request_with_submit_action,
+               description: 'Please take this',
+               creator: submitter,
+               target_package: target_package,
+               source_package: source_package)
+
+        create(:bs_request,
+               type: 'maintenance_release',
+               source_project: submitter.home_project)
+
+        create(:bs_request, type: 'role_change', description: "This request's description", creator: submitter)
+
+        create(:bs_request_with_maintenance_release_actions,
+               source_project: source_project,
+               state: :review)
+      end
+    end
+
+    context 'as a logged in user' do
+      before do
+        login(submitter)
+        Flipper.enable(:request_index)
+        get :index
+      end
+
+      it { expect(response).to have_http_status(:success) }
+      it { expect(response).to render_template('webui/request/index') }
+
+      it 'lists all the requests' do
+        expect(assigns(:bs_requests).count).to eq(BsRequest.count)
```

Comment:
I'm not quite sure what you are testing here...

Reply:
That the list is transparently returned by the route (no undesired default filtering).
But basically, nothing, it's a draft...